src/sync/incomingSyncUtils.js

```diff
@@ -244,6 +262,9 @@ export const sanityCheckIncomingRecord = (recordType, record) => {
         'cost_price',
```

Comment:
🤔 why does the `Store` have a cost and sell price?

Reply:
haha - I think it's just a weirdo diff

Reply:
that's good - I thought it was showing a huge gap in my understanding of the sync methods!